iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/dataclass/RelaySchedulePair.java

```diff
@@ -0,0 +1,58 @@
+package com.cgi.iec61850serversimulator.dataclass;
+
+public class RelaySchedulePair {
+    /**
+     * Class to store relay numbers and schedule numbers into a single object for
+     * EventDataListener HashMap and general refactoring of code
+     */
+
+    private int relayNr;
+    private int scheduleNr;
+
+    public RelaySchedulePair(int relayNr, int scheduleNr) {
+        this.relayNr = relayNr;
+        this.scheduleNr = scheduleNr;
+    }
+
+    public int getRelayNr() {
+        return this.relayNr;
+    }
+
+    public void setRelayNr(int relayNr) {
+        this.relayNr = relayNr;
+    }
```

Comment:
setRelayNr and the setScheduleNr should be removed. After initialising a RelaySchedulePair, it should not be possible to modify it.

Reply:
Done